core/kernel/persistence/smoothsql/search/driver/TaoSearchDriver.php

```diff
@@ -64,4 +64,17 @@ public function reserved($stringValue) {
         return $this->persistence->getPlatForm()->quoteIdentifier($stringValue);
     }
     
+    /**
+     * @inherit
+     */
+    public function random() {
```

Comment:
Could it be possible to have a random option binding in core_kernel_classes_Class::searchInstances() ?